src/main/java/seedu/address/logic/commands/AddNotesCommand.java

```diff
@@ -74,7 +74,7 @@ public CommandResult execute(Model model) throws CommandException {
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
 
-        return new CommandResult(generateSuccessMessage(editedPerson));
+        return new CommandResult(generateSuccessMessage(editedPerson), true);
```

Comment:
There is an issue where you do the following commands it will no clear the screen
1) `view 1`
2) `addNotes 1 n\Test`

This will not clear the contact details panel

Reply:
What do you mean by no clear the screen? Do you mean it doesn't get updated as shown below but only gets updated after doing `view 1` again?
![image](https://github.com/user-attachments/assets/5fe61477-0023-440c-87c7-daf445ceeb87)